model/ftn/wmesmfmd.ftn

```diff
@@ -271,6 +277,19 @@
       integer               :: natGridID
       logical               :: natGridIsLocal
       type(ESMF_RouteHandle):: n2eRH
+!
+! --- Mediator
+      logical        :: med_present = .false.
+      character(256) :: flds_scalar_name = ''
+      integer        :: flds_scalar_num = 0
+      integer        :: flds_scalar_index_nx = 0
```

Comment:
flds_scalar_index_nx and flds_scalar_index_nx are domain metadata that allows CMEPS to convert a mesh back to 2d space for diagnostic outputs.